README.md

```diff
@@ -125,14 +127,10 @@ Verify the setup, by [running tortillas](#running-tortillas). \
 You can now check out [Test specifications](#test-specifications) and add test specifications
 to your tests.
 
-### CI/CD
-[TODO]
-
```

Comment:
This is were I thought we could add a rough outline of how to set up gitlab and github CI/CD.

Reply:
Ups, revert!